docs/RUNNING-LOCALLY.md

```diff
@@ -20,7 +20,7 @@ Prerequisites
 - [Rimraf](https://github.com/isaacs/rimraf)
 - [Asar](https://github.com/electron/asar)
 - [NSIS v2.51](http://nsis.sourceforge.net/Main_Page) (v3.x won't work)
-- [Visual Studio Community 2013](https://www.visualstudio.com/en-us/news/vs2013-community-vs.aspx)
+- [Visual Studio Community 2013 (free)](https://www.microsoft.com/en-us/download/details.aspx?id=48129)
```

Comment:
@WasabiFan Nah, it's actually in the link-description, not in the actual URL ;) No link-breakage.

I was just suggesting it might be nice if it matched the visual style of the NSIS link, with the text-in-brackets _after_ the link-description, rather than as part of the link-description.